vanilla-framework/vanilla-brochure-theme
# Fix up docs

## Done

- Correct spelling of 'position'
- Include two missing mixin calls

## QA

- Run `gulp jekyll`
- Check example site works as expected
